### PR TITLE
Add Dawn 

### PR DESCRIPTION
**Pull request by miohtama**
Add space for Dawn
  
Etherscan: https://etherscan.io/token/0x580c8520deda0a441522aeae0f9f7a5f29629afa
CoinGecko: https://www.coingecko.com/en/coins/dawn-protocol
Website: https://dawn.org/
Twitter: https://twitter.com/DawnProtocol

### CODE REVIEW

File: skins/dawn.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// CSS is loaded always, even vieweing other spaces, so cannot load fonts without affecting the site performance a lot
+// @import url('https://fonts.googleapis.com/css2?family=Barlow+Condensed:ital,wght@0,300;0,400;0,500;0,700;1,400&display=swap');
+
+.dawn {
+    --primary-color: #fc8071;
+    --block-bg: white;
+    --text-color: #0b0521;
+    --heading-color: #140b38;
+    --bg-color: #f9fafb;
```
Comment (gokaiorg) [inline skins/dawn.scss:9]: Please use only theme variables. No extra custom styles are allowed.

File: spaces/domains.json [hunk 2]
```diff
@@ -43,6 +43,7 @@
   "vote.rari.capital": "rari",
   "vote.bloodyfi.org": "bloody",
   "vote.indexcoop.com": "index",
+  "governance.dawn.org": "dawn"
```
Comment (gokaiorg) [inline spaces/domains.json:46]: You miss a comma.
```suggestion
  "governance.dawn.org": "dawn",
```

File: spaces/dawn/index.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Dawn",
+  "network": "1",
+  "symbol": "DAWN",
+  "skin": "dawn",
```
Comment (gokaiorg) [inline spaces/dawn/index.json:5]: You will need to add domain field to make your subdomain work.
```suggestion
  "skin": "dawn",
  "domain": "governance.dawn.org",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: skins/dawn.scss (hunk 1 above)
Comment (miohtama) [reply skins/dawn.scss:9]: Did a copy from existing `aave.scss` where they had similar styles, so removing.

File: spaces/dawn/index.json (hunk 3 above)
Comment (miohtama) [reply spaces/dawn/index.json:5]: Thank you!

File: skins/dawn.scss (hunk 1 above)
Comment (miohtama) [reply skins/dawn.scss:6]: Removed any styles now.

File: spaces/domains.json (hunk 2 above)
Comment (miohtama) [reply spaces/domains.json]: Fixed.

File: spaces/dawn/index.json (hunk 3 above)
Comment (miohtama) [reply spaces/dawn/index.json:5]: Added domain - not sure in `domains.json` is also needed.

**Review by gokaiorg — CHANGES_REQUESTED** [review]
The images file size should not exceed 50KB. Please reduce their weight.
You will miss logo.png.

**Comment by miohtama** [comment]
> The images file size should not exceed 50KB. Please reduce their weight.
> You will miss logo.png.

Reduced space.png size with palettized PNG.

Also added `logo.png` (same as space.png) - not sure what is difference between these two.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**